### ISSUE TITLE
Assassin's Creed III Remastered (911400)

### ISSUE THREAD

**Issue by smo0z** [ValveSoftware/Proton#2482]
# Compatibility Report
- Name of the game with compatibility issues: Assassin's Creed III Remastered
- Steam AppID of the game: 911400

## System Information
- GPU: NVIDIA GeForce RTX 2080
- Driver/LLVM version: nvidia 418.49.04
- Kernel version: 5.0.4
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/smo0z/1337586d3e6ca33c3c7014f6e8e5931c

- Proton version: 4.2-1

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Both games only show a black screen.

## Reproduction
Try to start one of the games.

[steam-911400.log](https://github.com/ValveSoftware/Proton/files/3026828/steam-911400.log)

**Comment by HonkingGoose** [comment]
~@flibitijibibo Is this something you want labeled Xaudio2?~ It's labeled...
From the Proton log:
>7546.302:00b0:00b1:fixme:xaudio2:IXAudio2Impl_QueryInterface (0x3335d00)->({60d8dac8-5aa1-4e8e-b597-2f5e2883d484},0x32f2f0), not found

---

@zzhiyi Do you think this issue is Media Foundation related? There are mentions of mfplat.dll in the Proton log.

**Comment by kisak-valve** [comment]
Hello @smo0z, can you try disabling uplay's overlay if it's not already?

**Comment by smo0z** [comment]
Uplay overlay and in-game notifications are disabled. And because of the mfplat.dll I also tried the workaround of re2. without success.

**Comment by zzhiyi** [comment]
7546.475:00b0:00bb:trace:module:load_builtin_dll Trying built-in L"nvapi64.dll"
7546.475:00b0:00bb:warn:module:load_builtin_dll cannot open .so lib for builtin L"nvapi64.dll": /mnt/cdrive2/Games/Proton 4.2/dist/lib/wine/nvapi64.dll.so: cannot open shared object file: No such file or directory
7546.475:00b0:00bb:warn:module:load_dll Failed to load module L"C:\\windows\\system32\\nvapi64.dll"; status=c0000135

I am inclined to think this is the problem of missing nvapi64.dll on wine. For Nvidia graphics card users. You might want to try dxvk graphics card pci id spoofing.

**Comment by smo0z** [comment]
@zzhiyi Nope, changing dxgi.customDeviceId / dxgi.customVendorId does not change the behavior.

**Comment by smo0z** [comment]
I have now made a custom built (proton-tkg) and it works with it. I do not know if the log of it is helpful, but here it is: [steam-911400.log](https://github.com/ValveSoftware/Proton/files/3031844/steam-911400.log)

**Comment by howels** [comment]
@smo0z what did you have to change for your build to work?

**Comment by smo0z** [comment]
@howels [This](https://gist.github.com/smo0z/0c6068d2df7a61179e22cdbeb0982d0f) is my current config and with that it works.
